packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -32,14 +32,34 @@ const getConvoLimit = (options = {}) => {
 const Conversation = WebexPlugin.extend({
   namespace: 'Conversation',
 
+  /**
+   * @param {String} cluster the cluster containing the id
+   * @param {UUID} [id] the id of the conversation.
+   *  If empty, just return the base URL.
+   * @returns {String} url of the conversation
+   */
+  getUrlFromCluserId({cluster, id}) {
```

Comment:
Was this meant to be `getUrlFromClusterId`?

Reply:
You've never heard of a `cluser`? 😉
I will fix.